### PR TITLE
[source-s3] Bump python-cdk version to include CSV tab delimiter fix

### PR DESCRIPTION
**Pull request by blarghmatey**
The 0.70.1 release of the Airbyte Python CDK includes a fix for handling tab-delimited CSV files. This bumps the version dependency in source-s3 to incorporate that fix.

### CODE REVIEW

**Comment by blarghmatey** [comment]
@marcosmarxm

File: airbyte-integrations/connectors/source-s3/metadata.yaml [hunk 1]
```diff
@@ -10,7 +10,7 @@ data:
   connectorSubtype: file
   connectorType: source
   definitionId: 69589781-7828-43c5-9f63-8925b1c1ccc2
-  dockerImageTag: 4.5.10
+  dockerImageTag: 4.5.11
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-s3/metadata.yaml:13]: You need to update the docs changelog too if not tests will fail. Also Can you create a separated branch than the master? It make easier to run tests when I duplicated your PR.

**Review by marcosmarxm — CHANGES_REQUESTED** [review]

File: airbyte-integrations/connectors/source-s3/metadata.yaml (hunk 1 above)
Comment (blarghmatey) [reply airbyte-integrations/connectors/source-s3/metadata.yaml:13]: 👍

Comment (blarghmatey) [reply airbyte-integrations/connectors/source-s3/metadata.yaml:13]: I re-created the PR on a different base branch at https://github.com/airbytehq/airbyte/pull/36160

**Pull request closed without merging**